tools/image-chilly-swap

```diff
@@ -0,0 +1,4 @@
+# fuzz out mission specific pip reqs only to create a build with less security issues
```

Comment:
Does this work w/o doing "git mv" vs "mv"?   Eventually a GitHub Action to PR the diffs will happen.  Potentially it should just blow away env-frozen-old unless it all works out because env-frozen was already in git and env-frozen-old was not.  In any case,  IMHO env-frozen-old should not be saved.

Reply:
This tool is like if you want to do a manual chilly build it just swaps chilly for frozen and wasn't intended to necessarily be related to git. It seemed analogous to how the frozen build gets made but not checked in every time you run image-build (as in this is not a GH action)